src/Compilers/CSharp/Portable/CommandLine/CSharpCompiler.cs

```diff
@@ -360,9 +360,10 @@ protected override bool TryGetCompilerDiagnosticCode(string diagnosticId, out ui
 
         protected override ImmutableArray<DiagnosticAnalyzer> ResolveAnalyzersFromArguments(
             List<DiagnosticInfo> diagnostics,
-            CommonMessageProvider messageProvider)
+            CommonMessageProvider messageProvider,
+            out ImmutableArray<ISourceGenerator> generators)
```

Comment:
Feels odd to pass one set by return and the other by `out`. If we're changing the signature I'd pass both by `out` here to make it consistent.

Reply:
Probably prefer `out` but I'm probably not in the majority.